release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2468,6 +2468,47 @@ index a90ab14ff..f7f6f1a71 100644
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])
 
+[id="ocp-4-11-ran-known-issues"]
+== {product-title} 4.11 RAN known issues
+
+* After deploying a {sno} with the RAN DU profile, the `openvswitch: cpu_id mismatch with handler threads` Open vSwitch kernel logs are continuously generating. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
+
+* If secure boot is currently disabled and you attempt to enable it through ZTP, the installation will not begin the first time ZTP is started. When secure boot is enabled through ZTP, the boot options are configured before virtual CD is attached. Therefore, the first boot is booting into the existing HD with secure boot on. The installation then gets stuck because booting from CD is never triggered. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```

Comment:
```suggestion
* If secure boot is disabled and you attempt to enable it through ZTP, the installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The installation gets stuck because the system never boots from the CD. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```